tests/test_winnowing.py

```
@@ -77,6 +79,223 @@ def test_snippet_strip(self):
+
+    def test_windows_hash_normalization(self):
+        """Test that Windows hash properly normalizes different line endings."""
+        winnowing = Winnowing(debug=True)
+
+        # All these should produce the same Windows hash after normalization
+        content_lf = b'line1\nline2\nline3\n'
+        content_crlf = b'line1\r\nline2\r\nline3\r\n'
+        content_cr = b'line1\rline2\rline3\r'
+        content_mixed = b'line1\r\nline2\nline3\r'
+
+        hash_lf = winnowing.__calculate_opposite_line_ending_hash(content_lf)
```

Pylint (3.3.7)

[warning] Access to a protected member __calculate_opposite_line_ending_hash of a client class

(W0212)